### ISSUE TITLE
book-latex.ll crashes llines with malloc error on 'q'uitting selection

### ISSUE THREAD

**Issue by MichaelHendry** [lifelines/lifelines#395]
Menu option 'r'
Enter 'book-latex'
Browse for an individual
Enter 'q'

llines(23858,0x7fffac5c5380) malloc: *** error for object 0x7ff09cf12a40: pointer being freed was not allocated
*** set a breakpoint in malloc_error_break to debug                                                                                                                                             Abort trap: 6

**Comment by memmerto** [comment]
You can even skip the browse part - just hit enter at the browse prompt and it will immediately crash.

**Comment by MichaelHendry** [comment]
This has fixed this problem both on Mac 64-bit and Ubuntu. I thought it had failed at first, because I edited yacc.y and went through the usual sh build/autogen.sh->./configure->make->make install sequence.

I then tried deleting the lifelines directory and cloning a fresh set of files from GitHub. Once these had been edited, the recompiled versions were free of this failure.

I had run "make distclean" before recompiling the earlier files, but this seems not to have ensured that the changes in yacc.y were incorporated.
